app/views/contexts/list_collector_context.py

```diff
@@ -0,0 +1,76 @@
+from flask import url_for
+from flask_babel import lazy_gettext
+
+from app.views.contexts.context import Context
+from app.views.contexts.question import build_question_context
+
+
+class ListCollectorContext(Context):
+    @staticmethod
+    def get_item_title(
```

Comment:
If this method was made static, placeholder_renderer could be removed as an argument here and the class renderer be used instead.